backend/internal/handlers/review/service.go

```diff
@@ -0,0 +1,135 @@
+package review
+
+import (
+	"context"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ReviewDocument struct {
+	ID        primitive.ObjectID `bson:"_id,omitempty" json:"_id,omitempty"`
+	Rating    Rating             `bson:"rating" json:"rating"`
+	Picture   string             `bson:"picture" json:"picture"`
+	Content   string             `bson:"content" json:"content"`
+	Reviewer  Reviewer           `bson:"reviewer" json:"reviewer"`
+	Timestamp time.Time          `bson:"timestamp" json:"timestamp"`
+	Comments  []interface{}      `bson:"comments,omitempty" json:"comments,omitempty"`
```

Comment:
I think the structure of comments could be reworked a tad bit,
Currently the implementation looks like: 
```
[
 [{key1:k,val:v},
 {key2:k,val:v},
 {key3:k,val:v},], 

 [{key1:k,val:v},
 {key2:k,val:v},
 {key3:k,val:v},], 
]
```
where we are representing every key-value field in a comment in multiple objects, I would maybe consider a structure like this: 

```
[
    {k1:v1, k2:v2, k3:v3},
    {k1:v1, k2:v2, k3:v3},
    {k1:v1, k2:v2, k3:v3},
]
```
where each comment is represented as a single non-nested object (for now)